src/solidity/variable.js

```diff
@@ -0,0 +1,216 @@
+'use strict'
+
+module.exports = {
+  getType: function (variable, stateVars) {
+    var ret = {}
+    ret.originalType = variable.attributes.type
```

Comment:
Please document this function. What is the difference between `originalType` and `innerType`?